packages/components/src/TitleBlockZen/types.ts

```diff
@@ -1,7 +1,7 @@
 import { CompanyAvatarProps, GenericAvatarProps } from "~components/Avatar"
-import { ButtonProps, CustomButtonProps } from "~components/Button"
-import { MenuItemProps } from "~components/Menu"
 import { Select } from "~components/Select"
+import { MenuItemProps } from "~components/__actions__/v1"
+import { ButtonProps, CustomButtonProps } from "~components/__actions__/v2"
```

Comment:
Lol wild

Reply:
Wonder if we'll see these in the wild :P I think Doug's updated these so will fix these imports when that's merged in and I rebase off main :)